### PR TITLE
Add additionnal Janus Idp users

### PR DESCRIPTION
**Pull request by christophe-f**

### CODE REVIEW

File: cluster-scope/base/user.openshift.io/groups/janus-idp/group.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
-apiVersion: user.openshift.io/v1
+\apiVersion: user.openshift.io/v1
```
Comment (tumido) [inline cluster-scope/base/user.openshift.io/groups/janus-idp/group.yaml:1]: @christophe-f can you please fix this? Leading backslash would cause issues.

**Review by tumido — CHANGES_REQUESTED** [review]

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Comment by christophe-f** [comment]
oops. Typo fixed

**Pull request merged**